core/src/main/scala/com/indoorvivants/vcpkg/Vcpkg.scala

```diff
@@ -1,19 +1,14 @@
-package com.indoorvivants.vcpkg.sbt
+package com.indoorvivants.vcpkg
 
-import sbt._
-import sbt.Keys._
-import sbt.plugins.JvmPlugin
 import java.util.Arrays
 import scala.sys.process
+import java.io.File
 import java.nio.file.Files
 import java.util.stream.Collectors
-import com.indoorvivants.vcpkg.sbt.Platform.OS.Linux
-import com.indoorvivants.vcpkg.sbt.Platform.OS.MacOS
-import com.indoorvivants.vcpkg.sbt.Platform.OS.Unknown
-import com.indoorvivants.vcpkg.sbt.Platform.OS.Windows
+import Platform.OS._
 
 class Vcpkg(binary: File, installation: File, debug: String => Unit = _ => ()) {
-  import sys.process.*
+  import sys.process._
```

Comment:
what's the scalac option that allows to use * as a wildcard in Scala 2.13 ?

Reply:
`-source:3` I think?

Reply:
or `-Xsource:3`